### PR TITLE
Add tutorials for electrophysiology and fiber photometry

### PR DESCRIPTION
**Pull request by weiglszonja**
Add tutorials for electrophysiology and fiber photometry

### CODE REVIEW

File: src/constantinople_lab_to_nwb/schierek_embargo_2024/interfaces/schierek_embargo_2024_processedbehaviorinterface.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def add_to_nwbfile(
             if column in data:
                 data[column] = list(np.array(data[column]).astype(bool))
 
-        columns_to_add = data.keys()
+        columns_to_add = list(data.keys())
```
Comment (weiglszonja) [inline src/constantinople_lab_to_nwb/schierek_embargo_2024/interfaces/schierek_embargo_2024_processedbehaviorinterface.py:110]: rollback unrelated change
```suggestion
        columns_to_add = data.keys()
```

**Review by alessandratrapani — APPROVED** [review]
Looks good!
